frontend/packages/vkt/src/components/clerkExamEvent/overview/ClerkExamEventDetails.tsx

```diff
@@ -161,31 +170,129 @@ export const ClerkExamEventDetails = () => {
     }
   };
 
+  const registeredEnrollments = examEventDetails.enrollments.filter(
+    (enrollment) => enrollment.status === EnrollmentStatus.PAID
+  );
+  const unpaidEnrollments = examEventDetails.enrollments.filter(
+    (enrollment) => enrollment.status === EnrollmentStatus.EXPECTING_PAYMENT
+  );
+  const queuedEnrollments = examEventDetails.enrollments.filter(
+    (enrollment) => enrollment.status === EnrollmentStatus.QUEUED
+  );
+  const canceledEnrollments = examEventDetails.enrollments.filter(
+    (enrollment) => enrollment.status === EnrollmentStatus.CANCELED
+  );
+
   return (
-    <ClerkExamEventDetailsFields
-      examEvent={examEventDetails}
-      onComboBoxChange={(field: keyof ClerkExamEventBasicInformation) =>
-        handleComboBoxChange(field)
-      }
-      onDateChange={(
-        field: keyof Pick<
-          ClerkExamEventBasicInformation,
-          'date' | 'registrationCloses'
-        >
-      ) => handleDateChange(field)}
-      onCheckBoxChange={(field: keyof ClerkExamEventBasicInformation) =>
-        handleCheckBoxChange(field)
-      }
-      editDisabled={isViewMode}
-      topControlButtons={
-        <ControlButtons
-          onCancel={onCancel}
-          onEdit={onEdit}
-          onSave={onSave}
-          isViewMode={isViewMode}
-          hasRequiredDetails={hasRequiredDetails}
-        />
-      }
-    />
+    <>
+      <ClerkExamEventDetailsFields
+        examEvent={examEventDetails}
+        onComboBoxChange={(field: keyof ClerkExamEventBasicInformation) =>
+          handleComboBoxChange(field)
+        }
+        onDateChange={(
+          field: keyof Pick<
+            ClerkExamEventBasicInformation,
+            'date' | 'registrationCloses'
+          >
+        ) => handleDateChange(field)}
+        onCheckBoxChange={(field: keyof ClerkExamEventBasicInformation) =>
+          handleCheckBoxChange(field)
+        }
+        editDisabled={isViewMode}
+        topControlButtons={
+          <ControlButtons
+            onCancel={onCancel}
+            onEdit={onEdit}
+            onSave={onSave}
+            isViewMode={isViewMode}
+            hasRequiredDetails={hasRequiredDetails}
+          />
+        }
+      />
+      {registeredEnrollments.length > 0 && (
```

Comment:
Kaikki tästä alaspäin liittyy vain tilaisuuteen liittyvien ilmoittautumisten näyttämiseen sekä tietty sähköpostien kopiointiin ja excelin lataamiseen. Siirtäisin nämä tästä toisaalle jolloin tämän ClerkExamEventDetailsin toiminnoksi jäisi vain tutkintotilaisuuden päivittämisen hallinta. Nämä komponentit itsessään voisi kai rendata ClerkExamEventOverviewPagen alta? Alla olevat nappulat voisivat myös olla omassa komponentissaan.

Voisi tehdä muuten sinne minne näitä siirretään erillisen funktion, jolla luodaan nämä otsikko + taulukko kutakin ilmoittautumislistaa koskien.